### ISSUE TITLE
Add Ringmaster and LZ to an evaluation docker for diverse testing.

### ISSUE THREAD

**Issue by sethtroisi** [tensorflow/minigo#544]
https://github.com/tensorflow/minigo/blob/master/cluster/evaluator/evaluator_cc_wrapper.sh is really nice but it would be cool to test against LZ.

Could you make a dockerfile which contains LZ 
```
sudo apt install clinfo && clinfo

# Clone github repo
git clone https://github.com/gcp/leela-zero
cd leela-zero
git submodule update --init --recursive

# Install build depedencies
sudo apt install libboost-dev libboost-program-options-dev libboost-filesystem-dev opencl-headers ocl-icd-libopencl1 ocl-icd-opencl-dev zlib1g-dev

# Use stand alone directory to keep source dir clean
mkdir build && cd build
cmake ..
cmake --build .
./tests
curl -O https://zero.sjeng.org/best-network
./leelaz --weights best-network
```

and ringmaster
https://pypi.org/project/gomill/

I can then work with you to augment with a control file based on model params and some gsutil upload.

**Comment by amj** [comment]
ok how do we want to get the control file into the image?  You want it to fetch it at runtime, or build the control file into the image directly, or...?

**Comment by sethtroisi** [comment]
I would like to pass a gcs path and have it copy that control file locally to run it.

**Comment by amj** [comment]
base nvidia dockerfile is here:  https://gitlab.com/nvidia/cuda/blob/ubuntu16.04/9.0/base/Dockerfile

**Comment by sethtroisi** [comment]
Tracking some extra ideas:
fuego, patchi, ringmaster time_per_move...

**Comment by amj** [comment]
this issue described above was fixed via #676 .  Let's do further refinements in a new issue.
